src/SarifWeb/SarifWeb.csproj

```diff
@@ -43,7 +43,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
-    <PackageReference Include="Sarif.Multitool.Library" Version="3.0.0-beta1" />
+    <PackageReference Include="Sarif.Multitool.Library" Version="3.0.0" />
```

Comment:
><PackageReference Include="Sarif.Multitool.Library" Version="3.0.0" /> [](http://example.com/codeflow?start=4&length=70)

This hasn't fixed the alert. Can you also directly reference Newtonsoft 13.0.1 here?